### ISSUE TITLE
FIX_BUG_2: Fixed the issue with low accuracy during training and predict test

### ISSUE THREAD

**Issue by AHAD1405** [AHAD1405/classify_pinguing#10]
- **Fixed the issue with low accuracy during training and predict test**
-  figure out the wrong to calculate Softmax and calculate loss function
